tests/tests/datm_cdeps_lnd_era5_rst

```diff
@@ -0,0 +1,65 @@
+#
+#  DATM_CDEPS_NOAHMP_ERA5 test
+#
+
+export TEST_DESCR="DATM_CDEPS_NOAHMP_ERA5_RST - control restart"
+
+export CNTL_DIR="datm_cdeps_lnd_era5"
+
+export LIST_FILES="ufs.cpld.lnd.out.2011-01-01-43200.tile1.nc \
+                   ufs.cpld.lnd.out.2011-01-01-43200.tile2.nc \
+                   ufs.cpld.lnd.out.2011-01-01-43200.tile3.nc \
+                   ufs.cpld.lnd.out.2011-01-01-43200.tile4.nc \
+                   ufs.cpld.lnd.out.2011-01-01-43200.tile5.nc \
+                   ufs.cpld.lnd.out.2011-01-01-43200.tile6.nc"
+
+export_datm_cdeps
+
+export SYEAR=2011
+export SMONTH=01
+export SDAY=01
+export SHOUR=06
+export FHMAX=06
+export FHROT=0
+
+export DATM_IN_CONFIGURE=datm_in
+export MESH_ATM="INPUT_DATM/ERA5_mesh.nc"
+export atm_datamode="ERA5"
+export ATM_NX_GLB=1440
+export ATM_NY_GLB=721
+export EXPORT_ALL=.true.
+
+export DATM_STREAM_CONFIGURE=datm.streams.era5.IN
+export MESH_ATM_DATA="INPUT_DATM/ERA5_mesh.nc"
```

Comment:
@uturuncoglu It looks like you're in the process of getting things sync'd up. There have been **a lot** of changes at the script level since you originated the PR but I want to be sure things are kept consistent in the tests going forward. So a couple of things to keep in mind--

We don't want to use quotes in tests unless they are required---and they generally are not. 
We want to consistently use "INPUT" as the input directory, not "INPUT_DATM" etc.
Mesh names should contain only the actual mesh filename, not the path (ie, MESH_ATM=ERA_mesh.nc). It is easy to adjust the datm.in.IN to add the path

In terms of code updates, the other issue I see still is the use of fields still aliased to 'mean' instead of 'inst'. (I haven't had a chance yet to review the ncep note changes but these all need to be consistent.)

Thanks for all your hard work on the component addition!

Reply:
@DeniseWorthen Thanks for heads up. Yes, I am stying to sync the branches to be ready just after we have spack-stack 1.5.1 available. And, yes, I realized that all the inputs are now in INPUT directory. I am adjusting my tests to follow that convention. Once the tests are running again without any issue. I could also check the naming convention in the field dictionary and make changes if it is required.  Thanks again.

Reply:
@DeniseWorthen I think I could resolve this but let me know if you still seeing some issues in the tests